opensrp-immunization/src/main/java/org/smartregister/immunization/domain/VaccineTrigger.java

```diff
@@ -29,9 +30,12 @@ public static VaccineTrigger init(String vaccineCategory, JSONObject data) throw
             final String OFFSET = "offset";
             final String PREREQUISITE = "prerequisite";
             final String WINDOW = "window";
+            final String LMP = "lmp";
             if (data.getString(REFERENCE).equalsIgnoreCase(Reference.DOB.name())) {
-                return new VaccineTrigger(data.getString(OFFSET), data.has(WINDOW) ? data.getString(WINDOW) : null);
-            } else if (data.getString(REFERENCE).equalsIgnoreCase(Reference.PREREQUISITE.name())) {
+                return new VaccineTrigger(data.getString(OFFSET), data.has(WINDOW) ? data.getString(WINDOW) : null , Reference.DOB);
+            }else if (data.getString(REFERENCE).equalsIgnoreCase(Reference.LMP.name())) {
```

Comment:
this should be failing the CI build, since there's no space after the `}` is there a configuration problem w/the Codacity?